target/linux/ramips/image/mt7621.mk

```diff
@@ -222,6 +222,15 @@ define Device/buffalo_wsr-600dhp
 endef
 TARGET_DEVICES += buffalo_wsr-600dhp
 
+define Device/cudy-wr1300
```

Comment:
No, please not. Keep the underscore, this is OpenWrt policy.

Please add back the version with the correct SUPPORTED_DEVICES string.
In addition, please also add $(Device/dsa-migration) as found for other devices. This will force the user to use -F during upgrade, and is necessary anyway ...

Reply:
Changed back.